core/payment-driver/zksync/src/zksync/wallet.rs

```diff
@@ -100,11 +100,11 @@ pub async fn exit(msg: &Exit) -> Result<String, GenericError> {
 }
 
 pub async fn enter(msg: &Enter) -> Result<String, GenericError> {
```

Comment:
```suggestion
pub async fn enter(msg: Enter) -> Result<String, GenericError> {
```

Reply:
We can avoid some cloning if the message ownership is transferred to the function.